src/flownet/network_model/_generate_connections.py

```diff
@@ -319,7 +357,7 @@ def _create_entity_connection_matrix(
     aquifer_ends: List[Coordinate],
     max_distance_fraction: float,
     max_distance: float,
-    concave_hull_bounding_boxes: Optional[np.ndarray] = None,
+    concave_hull_bounding_boxes: Optional[List[np.ndarray]] = None,
```

Comment:
I changed `concave_hull_bounding_boxes` into a list of _concave_hull_bounding_boxes_. So the naming is not good anymore. Maybe it should be called something like _concave_hull_set_?

Reply:
eh, maybe not "set" if it is a "list" by the way.... then we should use a set ;) You choose 👍

Reply:
concave_hull_list it is ;)